bookmaker_titlepage.rb

```diff
@@ -3,81 +3,79 @@
 require 'fileutils'
 
 require_relative '../bookmaker/core/header.rb'
+require_relative '../bookmaker/core/metadata.rb'
```

Comment:
FYI, currently in most deployments, bookmaker_titlepage runs _before_ metadata_preprocessing (which is what gathers most of the metadata).